packages/odyssey-react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -23,6 +23,9 @@ export interface CheckboxProps
     ComponentPropsWithRef<"input">,
     "onChange" | "style" | "className" | "type" | "children"
   > {
+  /**
+   * Children are never rendered.
+   */
   children?: never;
```

Comment:
If this is the case, why include this at all?

Reply:
good question - I would say its a decision born out of our larger API design of providing "pass through" props to underlying DOM nodes as much as possible. There really is little harm in providing `children` when it isn't used but I thought we could add some finesse to components in our offering when they don't render children to make that clear at authorship time through TS language server integrations and even document it on our storybook site.